CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Bug Fixes
 
+* Fix typo: mangle properties key should be props
```

Comment:
Please revert this change as the `CHANGELOG` is generated and the PR isn't part of this particular release :)

Reply:
Thanks for the advice but I think this PR is not correct (it changed from 0.4.6 to 1.x)

Reply:
The `CHANGELOG` is still auto generated from commit messages and your PR isn't part of `v1.1.6` release, so this needs to be reverted please :)

Reply:
done